### DISCUSSION TITLE
Extension of the toast/snack bar for scheduling (such as Plugin.LocalNotification)

### DISCUSSION THREAD

**Discussion by true-perfect-code** [CommunityToolkit/Maui#2327] [New Feature Discussions]
I refer to the discussion https://github.com/CommunityToolkit/Maui/discussions/2326#discussioncomment-11181159 

It would be great if Toast/Snackbar could be extended so that a toast message can be scheduled for the future even if the application is closed.
It is important that the app is automatically reopened when the Toast/Snackbar message is clicked and that you can also pass an ID (which you set when creating the Toast/Snackbar message). This ID would then be used to display the correct data record from a table in the open app (can be an appointment from the calendar, for example).

The Plugin.LocalNotification is located here:
https://github.com/thudugala/Plugin.LocalNotification

Thanks
pc

**Comment by ghost** [comment]
Just for reference if somebody will be hitting this page trough search engines/AI.

[Maui - native support](**url**)
[Shiny.NET - local notifications](https://shinylib.net/client/notifications/)
